app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,162 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
+import { SlActionUndo } from "react-icons/sl";
+import {
+  TransformWrapper,
+  TransformComponent,
+  useControls,
+} from "react-zoom-pan-pinch";
 import { Crash } from "@/types/crashes";
 
-// import axios from "axios";
-// import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
-
 const CR3_DIAGRAM_BASE_URL = process.env.NEXT_PUBLIC_CR3_DIAGRAM_BASE_URL!;
 
+const ZoomResetControls = ({
+  setRotation,
+}: {
+  setRotation: (value: number) => void;
+}) => {
+  const { zoomIn, zoomOut, resetTransform } = useControls();
+
+  const handleReset = () => {
+    resetTransform();
+    setRotation(0);
+  };
+
+  // pixel pushing
+  const iconStyle = {
+    position: "relative" as const,
+    top: "-1px",
+    fontSize: "1.3em",
+  };
+
+  return (
+    <div
+      className="tools"
+      style={{
+        display: "flex",
+        justifyContent: "space-between",
+        width: "100%",
+        marginBottom: "5px",
+      }}
+    >
+      <ButtonGroup>
+        <Button
+          size="sm"
+          variant="primary"
+          onClick={() => zoomIn()}
+          title="Zoom In"
+        >
+          <BsZoomIn style={iconStyle} />
+        </Button>
+        <Button
+          size="sm"
+          variant="primary"
+          onClick={() => zoomOut()}
+          title="Zoom Out"
+        >
+          <BsZoomOut style={iconStyle} />
+        </Button>
+      </ButtonGroup>
+      <Button size="sm" variant="primary" onClick={handleReset} title="Reset">
+        <SlActionUndo style={iconStyle} />
+      </Button>
+    </div>
+  );
+};
+
+const RotateControls = ({
+  rotation,
+  setRotation,
+}: {
+  rotation: number;
+  setRotation: (value: number) => void;
+}) => {
+  const rotate = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setRotation(Number(event.target.value));
+  };
+
+  return (
+    <div className="rotate-controls" style={{ marginTop: "10px" }}>
+      <input
+        type="range"
+        min="-180"
+        max="180"
+        value={rotation}
+        className="form-control-range"
+        id="formControlRange"
+        onChange={rotate}
+        style={{ width: "70%" }}
+        title="Rotate Diagram"
+      />
```

Comment:
TIL we actually have a `Form.Range` [component](https://react-bootstrap.netlify.app/docs/forms/range) from react-boostrap. Would you mind swapping that in here?

Reply:
TIL x 2. Thanks!

https://github.com/cityofaustin/vision-zero/pull/1635/commits/1e533c6f275f1c822a4c2a77f8ec30ee8a89e376

Reply:
You bet - TIL x 2. https://github.com/cityofaustin/vision-zero/pull/1635/commits/1e533c6f275f1c822a4c2a77f8ec30ee8a89e376 

Thanks!